docs/index.rst

```diff
@@ -14,23 +14,86 @@ A dark gray titlebar is on the page. This confirms the theme template has been a
 Site icon
 ~~~~~~~~~
 
-The icon top-left has been rendered. This confirms the template can reference external resources.
+The icon top-left has been rendered. This confirms the template can reference external
+resources.
```

Comment:
This text should probably clarify there's 2 icons - the one in the titlebar, and the "site logo".